web/styles/recent_view.css

```diff
@@ -532,7 +532,7 @@
             display: flex;
             justify-content: center;
             align-items: center;
-            /* To accommodate zulip-icon-triple-users icon */
+            /* To accommodate conversation-partners-icon icon */
             width: 14px;
```

Comment:
Should the height and width here be converted to ems?

Reply:
Removed the width and height since they are not required here and the element can just take child's size.